src/Game.js

```diff
@@ -0,0 +1,83 @@
+import Car from './Car.js';
+import InputView from './InputView.js';
+import OutputView from './OutputView.js';
+import { splitCarNames } from './utils/splitCarNames.js';
+import { validateCarName, validateRoundCount } from './validate.js';
+
+class Game {
+  #round;
+
+  constructor() {
+    this.cars = [];
+    this.#round = 0;
+  }
+
+  #createCars(inputCarNames) {
+    const carNames = splitCarNames(inputCarNames);
+
+    return carNames.map((carName) => new Car(carName));
+  }
+
+  #setValidatedValues(roundCount, inputCarNames) {
+    this.#round = roundCount;
+    this.cars = this.#createCars(inputCarNames);
+  }
+
+  async start() {
+    const inputCarNames = await InputView.readCarNames();
+    validateCarName(inputCarNames);
+    const roundCount = await InputView.readRoundCount();
+    validateRoundCount(roundCount);
+
+    this.#setValidatedValues(roundCount, inputCarNames);
+
+    OutputView.printEmptyLine();
+    this.#progress();
+  }
+
+  #raceCars() {
+    this.cars.map((car) => {
+      car.runRound();
+      const [name, forwardCount] = [car.name, car.forwardCount];
+      OutputView.printCurrentCarInfo(name, forwardCount);
+    });
+  }
```

Comment:
이 부분에서 map을 사용하고 계시지만 return값이 없는 것 같아요! 이럴 땐 `forEach`가 조금 더 적절하지 않을까 하여 제안드립니다 👍 

```suggestion
#raceCars() {
  this.cars.forEach((car) => {
    car.runRound();
    const { name, forwardCount } = car;
    OutputView.printCurrentCarInfo(name, forwardCount);
  });
}
```